main/templates/main/login.html

```diff
@@ -0,0 +1,41 @@
+{% load crispy_forms_tags %}
+{% load static %}
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
+    <title>Hotails.com - Login!</title>
+</head>
+
+<body>
```

Comment:
Do note this will need to be changed once the navbar is implemented

Reply:
Hi @Omeramsc , why is that ? 
we don’t want that navbar will appear at login page .

Reply:
The Navbar should appear on all pages as it is a part of the base template. you should include logic to hide the buttons when the user is not logged in (or not of the right type)
there are exceptions to the rule, but I highly suggest you'll implement it like this on this project.